pkgs/by-name/_0/_010editor/package.nix

```
@@ -55,13 +56,11 @@ stdenv.mkDerivation (finalAttrs: {
       '';
       linuxInstall = ''
         mkdir -p $out/opt && cp -ar source/* $out/opt
-
-        # Unset wrapped QT plugins since they're already included in the package,
-        # else the program crashes because of the conflict
         makeWrapper $out/opt/010editor $out/bin/010editor \
-          --unset QT_PLUGIN_PATH
-
-        # Copy the icon and generated desktop file
+          --unset QT_PLUGIN_PATH \
+          --set TMPDIR /tmp \
+          --set QT_QPA_PLATFORM xcb \
+          --prefix LD_LIBRARY_PATH : ${qt5.qtbase}/lib
```

I think `TMPDIR` is already `/tmp` by default and this seems to work fine without `LD_LIBRARY_PATH`

```suggestion
          --set QT_QPA_PLATFORM xcb
```